erpnext/patches/v13_0/add_status.py

```diff
@@ -0,0 +1,9 @@
+import frappe, json
+
+def execute():
+	from erpnext.setup.setup_wizard.operations.install_fixtures import add_status_data
+
+	frappe.reload_doc("setup", "doctype", "Status")
+
+	if not frappe.db.a_row_exists("Status"):
+		add_status_data()
```

Comment:
```suggestion
	add_status_data()
```
- Since we are already checking for duplicates in `add_status_data`